recipes/llvm-openmp/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LLVMOpenMpConan(ConanFile):
+    name = "llvm-openmp"
+    short_name = "openmp"
+    description = ("The OpenMP (Open Multi-Processing) specification "
+                   "is a standard for a set of compiler directives, "
+                   "library routines, and environment variables that "
+                   "can be used to specify shared memory parallelism "
+                   "in Fortran and C/C++ programs. This is the LLVM "
+                   "implementation.")
+    license = "BSD-2-Clause"
```

Comment:
```suggestion
    license = "Apache-2.0"
```

https://github.com/llvm/llvm-project/blob/llvmorg-9.0.1/openmp/LICENSE.txt

Reply:
```suggestion
    license = ("Apache-2.0 WITH LLVM-exception")
```
It isn't Apache-2.0 but Apache 2.0 with LLVM exception

> The LLVM Project is under the Apache License v2.0 with LLVM Exceptions:

https://spdx.org/licenses/LLVM-exception.html

https://spdx.org/licenses/exceptions-index.html
>  Exceptions are added to a license using the License Expression operator, "WITH".

https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60
> Exception "WITH" Operator
Sometimes a set of license terms apply except under special circumstances. In this case, use the binary "WITH" operator to construct a new license expression to represent the special exception situation. A valid <license-expression> is where the left operand is a <simple-expression> value and the right operand is a <license-exception-id> that represents the special exception terms.  
For example, when the Bison exception is to be applied to GPL-2.0+, the expression would be: 
(GPL-2.0+ WITH Bison-exception-2.2)